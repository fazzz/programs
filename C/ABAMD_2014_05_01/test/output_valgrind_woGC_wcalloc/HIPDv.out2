/***********************************************/
steps            = 0  
total time       =      0.000 ps  
T_kelvin         = 0.00000000000000000000e+00 K  
toal_energy      = 2.34647697281844330064e+02 kcal/mol  
toal_vertial_energy      = 2.34647677419060158854e+02 kcal/mol  
kinetic_energy   = 1.98150929657759519387e+02 kcal/mol  
kinetic_energy_vertial   = 1.98630677969043644760e-05 kcal/mol  
potential_energy_real = 3.64967676240848177827e+01 kcal/mol  
potential_energy_vertial   = -3.97258519704671872550e-05 kcal/mol  
dihedral_energy  = 9.74399813645895918057e+00 kcal/mol  
elect_energy     = -7.60671960521487903861e+02 kcal/mol  
VDW_energy       = 1.28620722215825249441e+06 kcal/mol  
1_4_elect_energy = 0.00000000000000000000e+00 kcal/mol  
1_4_VDW_energy   = 0.00000000000000000000e+00 kcal/mol  
/***********************************************/
steps            = 10  
total time       =      0.010 ps  
T_kelvin         = 3.54014873851378312413e+120 K  
toal_energy      = 1.01461228211575516848e+231 kcal/mol  
toal_vertial_energy      = 1.01461228211575516848e+231 kcal/mol  
kinetic_energy   = 1.01461228211575516848e+231 kcal/mol  
kinetic_energy_vertial   = 8.76784105738307931460e+05 kcal/mol  
potential_energy_real = 5.25807499509325566578e+01 kcal/mol  
potential_energy_vertial   = 4.21465085395039295690e+03 kcal/mol  
dihedral_energy  = 2.58279804633069787201e+01 kcal/mol  
elect_energy     = -7.89327518309018500986e+02 kcal/mol  
VDW_energy       = 1.34478637609213497490e+06 kcal/mol  
1_4_elect_energy = 0.00000000000000000000e+00 kcal/mol  
1_4_VDW_energy   = 0.00000000000000000000e+00 kcal/mol  
/***********************************************/
steps            = 20  
total time       =      0.020 ps  
T_kelvin         =                      nan K  
toal_energy      =                      nan kcal/mol  
toal_vertial_energy      =                      nan kcal/mol  
kinetic_energy   =                      nan kcal/mol  
kinetic_energy_vertial   =                      nan kcal/mol  
potential_energy_real =                      nan kcal/mol  
potential_energy_vertial   =                      nan kcal/mol  
dihedral_energy  =                      nan kcal/mol  
elect_energy     =                      nan kcal/mol  
VDW_energy       =                      nan kcal/mol  
1_4_elect_energy = 0.00000000000000000000e+00 kcal/mol  
1_4_VDW_energy   = 0.00000000000000000000e+00 kcal/mol  
/***********************************************/
steps            = 30  
total time       =      0.030 ps  
T_kelvin         =                      nan K  
toal_energy      =                      nan kcal/mol  
toal_vertial_energy      =                      nan kcal/mol  
kinetic_energy   =                      nan kcal/mol  
kinetic_energy_vertial   =                      nan kcal/mol  
potential_energy_real =                      nan kcal/mol  
potential_energy_vertial   =                      nan kcal/mol  
dihedral_energy  =                      nan kcal/mol  
elect_energy     =                      nan kcal/mol  
VDW_energy       =                      nan kcal/mol  
1_4_elect_energy = 0.00000000000000000000e+00 kcal/mol  
1_4_VDW_energy   = 0.00000000000000000000e+00 kcal/mol  
/***********************************************/
steps            = 40  
total time       =      0.040 ps  
T_kelvin         =                      nan K  
toal_energy      =                      nan kcal/mol  
toal_vertial_energy      =                      nan kcal/mol  
kinetic_energy   =                      nan kcal/mol  
kinetic_energy_vertial   =                      nan kcal/mol  
potential_energy_real =                      nan kcal/mol  
potential_energy_vertial   =                      nan kcal/mol  
dihedral_energy  =                      nan kcal/mol  
elect_energy     =                      nan kcal/mol  
VDW_energy       =                      nan kcal/mol  
1_4_elect_energy = 0.00000000000000000000e+00 kcal/mol  
1_4_VDW_energy   = 0.00000000000000000000e+00 kcal/mol  
/***********************************************/
steps            = 50  
total time       =      0.050 ps  
T_kelvin         =                      nan K  
toal_energy      =                      nan kcal/mol  
toal_vertial_energy      =                      nan kcal/mol  
kinetic_energy   =                      nan kcal/mol  
kinetic_energy_vertial   =                      nan kcal/mol  
potential_energy_real =                      nan kcal/mol  
potential_energy_vertial   =                      nan kcal/mol  
dihedral_energy  =                      nan kcal/mol  
elect_energy     =                      nan kcal/mol  
VDW_energy       =                      nan kcal/mol  
1_4_elect_energy = 0.00000000000000000000e+00 kcal/mol  
1_4_VDW_energy   = 0.00000000000000000000e+00 kcal/mol  
/***********************************************/
steps            = 60  
total time       =      0.060 ps  
T_kelvin         =                      nan K  
toal_energy      =                      nan kcal/mol  
toal_vertial_energy      =                      nan kcal/mol  
kinetic_energy   =                      nan kcal/mol  
kinetic_energy_vertial   =                      nan kcal/mol  
potential_energy_real =                      nan kcal/mol  
potential_energy_vertial   =                      nan kcal/mol  
dihedral_energy  =                      nan kcal/mol  
elect_energy     =                      nan kcal/mol  
VDW_energy       =                      nan kcal/mol  
1_4_elect_energy = 0.00000000000000000000e+00 kcal/mol  
1_4_VDW_energy   = 0.00000000000000000000e+00 kcal/mol  
/***********************************************/
steps            = 70  
total time       =      0.070 ps  
T_kelvin         =                      nan K  
toal_energy      =                      nan kcal/mol  
toal_vertial_energy      =                      nan kcal/mol  
kinetic_energy   =                      nan kcal/mol  
kinetic_energy_vertial   =                      nan kcal/mol  
potential_energy_real =                      nan kcal/mol  
potential_energy_vertial   =                      nan kcal/mol  
dihedral_energy  =                      nan kcal/mol  
elect_energy     =                      nan kcal/mol  
VDW_energy       =                      nan kcal/mol  
1_4_elect_energy = 0.00000000000000000000e+00 kcal/mol  
1_4_VDW_energy   = 0.00000000000000000000e+00 kcal/mol  
/***********************************************/
steps            = 80  
total time       =      0.080 ps  
T_kelvin         =                      nan K  
toal_energy      =                      nan kcal/mol  
toal_vertial_energy      =                      nan kcal/mol  
kinetic_energy   =                      nan kcal/mol  
kinetic_energy_vertial   =                      nan kcal/mol  
potential_energy_real =                      nan kcal/mol  
potential_energy_vertial   =                      nan kcal/mol  
dihedral_energy  =                      nan kcal/mol  
elect_energy     =                      nan kcal/mol  
VDW_energy       =                      nan kcal/mol  
1_4_elect_energy = 0.00000000000000000000e+00 kcal/mol  
1_4_VDW_energy   = 0.00000000000000000000e+00 kcal/mol  
/***********************************************/
steps            = 90  
total time       =      0.090 ps  
T_kelvin         =                      nan K  
toal_energy      =                      nan kcal/mol  
toal_vertial_energy      =                      nan kcal/mol  
kinetic_energy   =                      nan kcal/mol  
kinetic_energy_vertial   =                      nan kcal/mol  
potential_energy_real =                      nan kcal/mol  
potential_energy_vertial   =                      nan kcal/mol  
dihedral_energy  =                      nan kcal/mol  
elect_energy     =                      nan kcal/mol  
VDW_energy       =                      nan kcal/mol  
1_4_elect_energy = 0.00000000000000000000e+00 kcal/mol  
1_4_VDW_energy   = 0.00000000000000000000e+00 kcal/mol  
