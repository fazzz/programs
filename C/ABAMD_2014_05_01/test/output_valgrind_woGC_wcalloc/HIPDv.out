0 6.42733019097001990303e+05 1.98150929657759519387e+02 1.98630677969043644760e-05 -3.97258519704671872550e-05 6.42931170006796950474e+05 9.99995240352815839557e-01 0.00000000000000000000e+00
10 6.72024352267376263626e+05 1.01461228211575516848e+231 8.76784105738307931460e+05 4.21465085395039295690e+03 1.01461228211575516848e+231 2.01826245706610251202e+219 3.54014873851378312413e+120
20                      nan                      nan                      nan                      nan                      nan                      nan                      nan
30                      nan                      nan                      nan                      nan                      nan                      nan                      nan
40                      nan                      nan                      nan                      nan                      nan                      nan                      nan
50                      nan                      nan                      nan                      nan                      nan                      nan                      nan
60                      nan                      nan                      nan                      nan                      nan                      nan                      nan
70                      nan                      nan                      nan                      nan                      nan                      nan                      nan
80                      nan                      nan                      nan                      nan                      nan                      nan                      nan
90                      nan                      nan                      nan                      nan                      nan                      nan                      nan
